src/MongoFramework/Linq/LinqExtensions.cs

```diff
@@ -53,7 +53,7 @@ public static string ToQuery(this IQueryable queryable)
 			return queryable.Where(expression);
 		}
 
-		private static IQueryable<TEntity> WhereFilter<TEntity>(this IQueryable<TEntity> queryable, Func<FilterDefinitionBuilder<TEntity>, FilterDefinition<TEntity>> queryFilter)
+		public static IQueryable<TEntity> WhereFilter<TEntity>(this IQueryable<TEntity> queryable, Func<FilterDefinitionBuilder<TEntity>, FilterDefinition<TEntity>> queryFilter)
```

Comment:
Make this `internal` rather than `public`

Reply:
Done.